ui/scss/component/_videojs.scss

```diff
@@ -225,3 +225,22 @@ $control-bar-icon-size: 0.8rem;
     }
   }
 }
+
+// larger than default spinner
+.vjs-loading-spinner {
+  margin: -49px 0 0 -37px;
+  border-radius: 100px;
+}
```

Comment:
Uneven margins -- high chance it will not work in some resolutions.

It's off-centered in Firefox Android on a 1080x2400 device (works fine in production):

![image](https://user-images.githubusercontent.com/64950861/148188148-c007bd99-ce6e-40af-bb42-4f3e9ac7e131.png)